lib/octokit/client/releases.rb

```diff
@@ -121,9 +121,9 @@ def delete_release_asset(repo, asset_id, options = {})
       #
       # @param repo [Integer, String, Repository, Hash] A GitHub repository
       # @param tag [String] The tag of the release
-      # @return [Sawyer::Resource] A single release by tag
+      # @return [Sawyer::Resource] A single release
       # @see https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name
-      def release_by_tag(repo, tag, options = {})
+      def tag_release(repo, tag, options = {})
```

Comment:
This name is not super clear, could we revert this? Or find an alternative? This makes it feel like we're tagging a release.